.git2gus/config.json

```diff
@@ -25,11 +25,11 @@
     "area:xml": "a1aB0000000BPyKIAW"
   },
   "issueTypeLabels": {
-    "type:feature": "",
+    "feature": "USER STORY",
     "type:bug": "BUG P3",
```

Comment:
it will be good to have more granular types here that lets us create p0 p1 p2 p3 bugs. 
also can you explain why did you take out the word "type"?

Reply:
something like
    "type:bug-p0": "BUG P0",
    "type:bug-p1": "BUG P1",
    "type:bug-p2": "BUG P2",

default can be P3 as it exists today

Reply:
These are the existing labels that we can tag Github issues with.  We tag features with the `feature` tag - there is no `type:feature` tag in our options.

As for bugs, we currently have the `type:bug` tag, which automatically creates a P3 bug.  I can make 3 new tags for P0 bug, P1 bug, and P2 bug and then update this PR to wire them to GUS.